modules/core_project_factory/main.tf

```diff
@@ -169,6 +170,18 @@ resource "google_project_services" "project_services_authority" {
   depends_on = [google_project.main]
 }
 
+resource "google_project_service" "iam_credentials_api" {
```

Comment:
Instead of a separate service block, we should inject this into the `activate_apis`. (In particular, this would break in the case of authoritatively setting the project services.)

Reply:
This has been fixed in the latest commit and is injected in the locals block.